Bottom: 6528c6bbb54b961c1d88354c6c20dfb9ba83de31
Top:    c8b7f4a171d8346339aab469f6ea2ceceb9cd715
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 15:50:23 +0100

Refresh of hid-gtk-don-t-use-gtk_widget_s

---

diff --git a/src/hid/gtk/gui-config.c b/src/hid/gtk/gui-config.c
index 0833486..851571a 100644
--- a/src/hid/gtk/gui-config.c
+++ b/src/hid/gtk/gui-config.c
@@ -2174,7 +2174,7 @@ ghid_config_window_show (void)
   gtk_box_pack_start (GTK_BOX (main_vbox), hbox, FALSE, FALSE, 0);
 
   button = gtk_button_new_from_stock (GTK_STOCK_OK);
-  GTK_WIDGET_SET_FLAGS (button, GTK_CAN_DEFAULT);
+  gtk_widget_set_can_default (button, TRUE);
   g_signal_connect (G_OBJECT (button), "clicked",
 		    G_CALLBACK (config_close_cb), NULL);
   gtk_box_pack_start (GTK_BOX (hbox), button, TRUE, TRUE, 0);
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 9c22ea2..06eab4d 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1271,7 +1271,7 @@ ghid_build_pcb_top_window (void)
    * enter and button press callbacks grab focus to be sure we have it
    * when in the drawing_area.
    */
-  GTK_WIDGET_SET_FLAGS (gport->drawing_area, GTK_CAN_FOCUS);
+  gtk_widget_set_can_focus (gport->drawing_area, TRUE);
 
   gtk_box_pack_start (GTK_BOX (hbox), gport->drawing_area, TRUE, TRUE, 0);
